### PR TITLE
Add Yohai Magen to AUTHORS.md

### PR DESCRIPTION
**Pull request by yohaimagen**
Hi, I'm opening this pull request in order to add myself to ` AUTHERS.md`.
I am pleased to join this wonderful team.
A bit about myself, my name is Yohai Magen and I am from Israel. I'm in my fourth year of a geophysics P.h.D in seismology/geodesy, working on earthquake physics with geodetic data sets mostly InSAR and GNSS, and some satellite optical imagery. I was a GMT user to produce most of my research figures, and mostly shifted to PyGMT in the past year or so. 
Hope to contribute to this great project :).

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -18,5 +18,6 @@ The following people have contributed code and/or documentation to the project
 * [Meghan Jones](https://github.com/meghanrjones) | [0000-0003-0180-8928](https://orcid.org/0000-0003-0180-8928) | University of Hawai'i at Mānoa
 * [Michael Grund](https://github.com/michaelgrund) | [0000-0001-8759-2018](https://orcid.org/0000-0001-8759-2018) | Innoplexia GmbH
 * [Tyler Newton](http://www.tnewton.com/) | [0000-0002-1560-6553](https://orcid.org/0000-0002-1560-6553) | University of Oregon
+* [Yohai Magen](https://github.com/yohaimagen) | [0000-0002-4892-4013](https://orcid.org/0000-0002-4892-4013) | Tel Aviv University
 * [Wei Ji Leong](https://github.com/weiji14) | [0000-0003-2354-1988](https://orcid.org/0000-0003-2354-1988) | Victoria University of Wellington
 * [William Schlitzer](https://github.com/willschlitzer) | [0000-0002-5843-2282](https://orcid.org/0000-0002-5843-2282) | Unaffiliated
```
Comment (core-man) [inline AUTHORS.md:23]: Sort alphabetically based on the first name?
```suggestion
* [William Schlitzer](https://github.com/willschlitzer) | [0000-0002-5843-2282](https://orcid.org/0000-0002-5843-2282) | Unaffiliated
* [Yohai Magen](https://github.com/yohaimagen) | [0000-0002-4892-4013](https://orcid.org/0000-0002-4892-4013) | Tel Aviv University
* [Wei Ji Leong](https://github.com/weiji14) | [0000-0003-2354-1988](https://orcid.org/0000-0003-2354-1988) | Victoria University of Wellington
```

**Review by weiji14 — APPROVED** [review]
Welcome to the team! You'll be joining quite a few seismologists here :grin: @core-man, do you mind if we get Yohai's name onto the v0.4.1 release (specifically the BibTeX)? If so I can merge this PR in right now on a second approval.

Cc @seisman / @meghanrjones to add our new member to https://github.com/orgs/GenericMappingTools/teams/python-contributors.

**Review by core-man — APPROVED** [review]
> Welcome to the team! You'll be joining quite a few seismologists here  @core-man, do you mind if we get Yohai's name onto the v0.4.1 release (specifically the BibTeX)? If so I can merge this PR in right now on a second approval.

Great~

**Review by michaelgrund — APPROVED** [review]
Great!

**Comment by seisman** [comment]
> Welcome to the team! You'll be joining quite a few seismologists here 😁 @core-man, do you mind if we get Yohai's name onto the v0.4.1 release (specifically the BibTeX)? If so I can merge this PR in right now on a second approval.
> 
> Cc @seisman / @meghanrjones to add our new member to [github.com/orgs/GenericMappingTools/teams/python-contributors](https://github.com/orgs/GenericMappingTools/teams/python-contributors).

I just sent the invitation to @yohaimagen.

**Comment by meghanrjones** [comment]
Welcome to the team @yohaimagen!

**Comment by weiji14** [comment]
I've just added you to the [PyGMT ResearchGate project page](https://www.researchgate.net/project/PyGMT-A-Python-interface-for-the-Generic-Mapping-Tools) too Yohai :wink: If you want, we can also invite you to the PyGMT Dev Slack channel.

P.S. Team, maybe the onboarding checklist at https://www.researchgate.net/project/PyGMT-A-Python-interface-for-the-Generic-Mapping-Tools should be split into 2 sections, one for contributors and one for maintainers?

**Comment by meghanrjones** [comment]
> P.S. Team, maybe the onboarding checklist at https://www.researchgate.net/project/PyGMT-A-Python-interface-for-the-Generic-Mapping-Tools should be split into 2 sections, one for contributors and one for maintainers?

Agree about adding info for contributors separately here https://www.pygmt.org/dev/maintenance.html#onboarding-access-checklist

**Pull request merged**